src/anonymizer.js

```diff
@@ -2,7 +2,7 @@ import log from "./log.js";
 import { DicomMetaDictionary } from "./DicomMetaDictionary.js";
 import { Tag } from "./Tag.js";
 
-var tagNamesToEmpty = [
+export var tagNamesToEmpty = [
```

Comment:
I'm thinking there could be side effects here - one piece of code modifies the anonymization feature and then later it might be used by another piece of code that expects it to be in the default state.

Maybe instead we a a method to return a copy of the list and then pass in the lists as optional arguments to `cleanTags` (with the internal defaults being used if not specified).

Reply:
ok thx. I will work on it soon

Reply:
done